src/__experimental__/components/switch/switch.style.js

```diff
@@ -37,7 +38,20 @@ const StyledSwitch = styled.div`
       margin-bottom: 8px;
       margin-right: 100%;
 
-      ${labelWidth && `
+      ${StyledValidationIcon} {
+        display: inline-block;
+        padding: 1px;
+        margin-top: 0;
+        margin-left: 8px;
+        margin-bottom: 0;
+        vertical-align: middle;
+      }
+
+      ${hasInfo && css`color: ${theme.colors.info};`}
```

Comment:
You only need to use `css` if you have a function inside the template literal. It's safe to remove the call to `css` on the next 3 lines too.

https://www.styled-components.com/docs/api#css

Reply:
Updated in commit 02dd7594e